packages/atomic/src/components/facets/atomic-date-facet/atomic-date-facet.tsx

```diff
@@ -104,7 +113,9 @@ export class AtomicDateFacet implements InitializableComponent, BaseFacetState {
 
     return (
       <FacetValue
-        label={` ${item.start}-${item.end}`}
+        label={`${dayjs(item.start).format(this.dateFormat)} - ${dayjs(
+          item.end
+        ).format(this.dateFormat)}`}
```

Comment:
I see some duplication between the date facet and breadcrumb manager. Could we extract the logic into a central date formatting function? This will help ensure both places remain in-sync.